blame.go

```diff
@@ -30,66 +30,36 @@ func Blame(c *object.Commit, path string) (*BlameResult, error) {
 	// The file to blame is identified by the input arguments:
 	// commit and path. commit is a Commit object obtained from a Repository. Path
 	// represents a path to a specific file contained into the repository.
-	//
```

Comment:
It would be good to keep (an updated) high-level description of the implementation.

Reply:
Will do (I was planning to)